finalfrontier-utils/src/util.rs

```diff
@@ -233,22 +236,25 @@ impl DepembedsApp {
             Arg::with_name(CONTEXT_DISCARD)
                 .long("context_discard")
                 .value_name("CONTEXT_THRESHOLD")
-                .help("Context discard threshold (default: 1e-4)")
-                .takes_value(true),
+                .help("Context discard threshold")
+                .takes_value(true)
+                .default_value("1e-4"),
         )
         .arg(
             Arg::with_name(CONTEXT_MINCOUNT)
                 .long("context_mincount")
                 .value_name("CONTEXT_FREQ")
-                .help("Context mincount (default: 5)")
-                .takes_value(true),
+                .help("Context mincount")
+                .takes_value(true)
+                .default_value("5"),
         )
         .arg(
             Arg::with_name(DEPENDENCY_DEPTH)
                 .long("dependency_depth")
                 .value_name("DEPENDENCY_DEPTH")
-                .help("Dependency depth (default: 2)")
-                .takes_value(true),
+                .help("Dependency depth")
+                .takes_value(true)
+                .default_value("2"),
```

Comment:
There was a mismatch between the help message and the `unwrap_or` below. The default value should probably be 1, since that's what Levy & Goldberg describe in their paper.

Reply:
Ah, indeed! Fixed now.